fastchat/serve/gradio_web_server.py

```diff
@@ -771,7 +785,7 @@ def build_about():
 def build_single_model_ui(models, add_promotion_links=False):
     promotion = (
         """
-- | [GitHub](https://github.com/lm-sys/FastChat) | [Dataset](https://github.com/lm-sys/FastChat/blob/main/docs/dataset_release.md) | [Twitter](https://twitter.com/lmsysorg) | [Discord](https://discord.gg/HSWAKCrnFx) |
+- | [GitHub](https://github.com/lm-sys/FastChat) | [Twitter](https://twitter.com/lmsysorg) | [Discord](https://discord.gg/HSWAKCrnFx) |
```

Comment:
why do we remove dataset link?

Reply:
ah this was for single model chat so I thought it didnt make sense to put the dataset of battles here. I can add it back in though

Reply:
fixed!